doc/userguide/rules/datasets.rst

```diff
@@ -183,6 +227,10 @@ md5
   in the file as hex encoded string
 sha256
   in the file as hex encoded string
+ipv4
+  in the file as string
+ip
+  in the file as string, it can be IPv6 or IPv4 address (standard notation or IPv4 in IPv6 one)
```

Comment:
"IPv4 in IPv6 one" -> not sure what is meant here?

Reply:
This is notation like 192.168.1.2 is ::FFFF:192.168.1.2 (https://www.ibm.com/docs/en/zos/2.2.0?topic=addresses-ipv4-mapped-ipv6)

Reply:
Maybe we can add a note/explanation, for those who are unfamiliar with this notation?